src/eyebot_controller/lcd/src/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <Arduino.h>
```

Comment:
Add comments to each line / section explaining what it does. Both for readability and for you to ensure you understand what is happening.